doc/authors.md

```diff
@@ -2,13 +2,13 @@ Zcash Contributors
 ==================
 
 Jack Grigg (1297)
-Kris Nuttycombe (622)
+Kris Nuttycombe (626)
 Simon Liu (460)
 Sean Bowe (389)
-Daira Hopwood (379)
+Daira Hopwood (382)
```

Comment:
I have 43 commits in this release (9 as Daira Hopwood, 34 as Daira Emma Hopwood). The commit counts are definitely not being calculated correctly &mdash; maybe it's only counting merge commits?

Reply:
This one is a known bug: if you appear in the history as multiple names, their counts are computed separately, and then IIRC the last one is the one that gets applied here. This also needs regenerating from the history after the bug is corrected.

Reply:
Yeah, I noticed the same for Greg; he has 81 commits in this release, but only gets credit for 1!